.travis.yml

```diff
@@ -58,9 +58,18 @@ matrix:
      os: osx
    - env: GHCVER=8.0.1 SCRIPT=bootstrap
      os: osx
+
+   - env: GHCVER=via-stack SCRIPT=stack STACKAGE_RESOLVER=lts
+     os: linux
+     addons:
+        apt:
+            packages:
+                - libgmp-dev
+
   allow_failures:
    - env: GHCVER=head
      sudo: required
+   - env: GHCVER=via-stack SCRIPT=stack STACKAGE_RESOLVER=lts
```

Comment:
I think we can just remove the `allow_failures` section since we're not testing with GHC HEAD.